pkg/devfile/adapters/kubernetes/component/adapter.go

```diff
@@ -391,8 +392,25 @@ func (a Adapter) createOrUpdateComponent(componentExists bool, ei envinfo.EnvSpe
 		return err
 	}
 
+	// Get ServiceBinding Secret name for each Link
+	var sbSecrets []string
+	for _, link := range ei.GetLink() {
+		sb, err := a.Client.GetDynamicResource(kclient.ServiceBindingGroup, kclient.ServiceBindingVersion, kclient.ServiceBindingResource, link.Name)
+		if err != nil {
+			return err
+		}
+		secret, found, err := unstructured.NestedString(sb.Object, "status", "secret")
+		if err != nil {
+			return err
+		}
+		if !found {
+			return fmt.Errorf("unable to find secret in ServiceBinding %s", link.Name)
```

Comment:
```suggestion
			return fmt.Errorf("unable to find secret in ServiceBinding %q", link.Name)
```
In an attempt to make odo errors more readable on the CLI.